taipy/core/_orchestrator/_dispatcher/_job_dispatcher.py

```diff
@@ -29,9 +29,9 @@ class _JobDispatcher(threading.Thread):
     """Manages job dispatching (instances of `Job^` class) on executors."""
 
     _STOP_FLAG = False
-    _dispatched_processes: Dict = {}
```

Comment:
Oh so we do not use this anymore. That's good to know

Reply:
Well, that's the purpose of my proposal. It's up to us to reject it.